### ISSUE TITLE
CEN1 has some data from logger test prior installation

### ISSUE THREAD

**Issue by BaptisteVandecrux** [GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues#128]
![CEN1_0](https://github.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/assets/35140661/1c85f45e-c8e6-4bfb-a29f-2b59469dd94e)

Fixed in 
[d518a51](https://github.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/commit/d518a51424952e1cffc2ea4c91dfbed690ab0ba0)

**Comment by PennyHow** [comment]
Should be resolved once this is merged into our aws-l0 repo: https://geusgitlab.geus.dk/glaciology-and-climate/promice/aws-l0/-/merge_requests/53

**Comment by BaptisteVandecrux** [comment]
fixed in dataverse release V13
![billede](https://github.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/assets/35140661/76d1cd47-0687-4958-827d-95bfb4616a2a)
